### DISCUSSION TITLE
Rollup #1: May 21, 2021 Community Call Topics

### DISCUSSION THREAD

**Discussion by wpmonty** [BanklessDAO/bankless-dao-website#45] [Q&A]
The community organizers are looking for a representative to voice the guild for the community call on Friday.

Specifically, they want us to relay updates, needs, plans, and anything else to the community at large.

### Please post topics below that you would like included in the community call.

For example:

I propose we surface the Onboard project (still in its infancy, but shed some light on its formulation and timeline).
I propose we surface @Tgriffith3433's Bankless NFT auction house project and give them a few minutes to speak.
I propose we surface @MrPackman's persona research, especially how it relates to building products.

I also want to spend some time managing expectations and why we may seem to be moving slower than some other guilds.

What else should we bring up?

**Comment by Icedcool** [comment]
I propose we surface the analytics that Air and Paul are working on, if that isn't already on the agenda. They've been up to great work.

As to the Dev guild moving slow, personally I'm seeing the DAO as a media/informational DAO(at least right now), and it seems that the dev is in a support position of that initiative... and it doesn't seem like a lot of dev work yet?

That said, I am still cooking on some dev initiatives.

Wasn't there also talk of a website redesign?

**Reply by PaulApivat** [reply]
Thanks you @Icedcool , we'll be happy to provide an update on the membership analytics we've been working on.

**Reply by PaulApivat** [reply]
Here's some context for [Membership Categories / Analytics](https://forum.bankless.community/t/bank-member-categories-dashboard/638), we can elaborate more in the community call.

**Reply by wpmonty** [reply]
I think @davidhoffman wanted to kick off some redesign project, but I haven't seen any movement. We also had a member build out a new site design of his own volition, but no traction since that initial post. 

Fair point to the purpose of the guild relative to the dao - I'm sure we will have work to do soon though

**Comment by MrPackman4019** [comment]
Mentioned this in discord, the user research/personas will be covered by the marketing guild recap which I am coordinating. Not saying don't mention it with the dev guild as it does tie in here as well, just may have covered already or it will be covered in more detail when marketing is called up.
